### ISSUE TITLE
edit model hyperparameter configuration

### ISSUE THREAD

**Issue by WidiaPrasetia** [ultralytics/hub#834]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and found no similar feature requests.

### Description

After everything has been configured and then saved. maybe when the user wants to edit the train model configuration, it can be done. because there is no such menu feature.
then in the custom features you might be able to add several other hyperparameters that can be customized, such as dropout for example and several other features that don't exist.

### Use case

when you have finished configuring the train model and then save it. but there was something I forgot, then I wanted to edit it, for example patience which I should have configured to 10, but it was still 100 because I forgot to configure it. Well, this is where the configuration edit feature is needed

**Comment by WidiaPrasetia** [comment]
Thank you for responding to the issue I created.

Yes, there really isn't any dropout. does he have another name?

To make it clearer, I have attached the proof below.

Thank You
Warm regards
![image](https://github.com/user-attachments/assets/fe0deb76-599b-4e4f-b17f-2ab3f3c657bd)

**Comment by sergiuwaxmann** [comment]
@WidiaPrasetia Yes, I can confirm it is missing. Our team will introduce it in the next release. I will keep you updated.

**Comment by WidiaPrasetia** [comment]
thankyou sir for your response, good luck!
I will always wait for the update

**Comment by sergiuwaxmann** [comment]
@WidiaPrasetia `dropout` is available but only for the Classify task.
![SCR-20240913-qlf](https://github.com/user-attachments/assets/a68bb007-6311-462d-a6b1-0b547f3c9a4e)

_Dropout rate for regularization in classification tasks, preventing overfitting by randomly omitting units during training._
(Read more here: https://docs.ultralytics.com/usage/cfg/?h=train+settings#train-settings)

Related to the Excel export - we don't have such option at the moment.

**Comment by WidiaPrasetia** [comment]
then for the detect task is there not yet?

**Comment by WidiaPrasetia** [comment]
Sorry, this deviates from the issue I created. can you help me in this case?

I have finished training the model with 100 epochs. but why does it still appear like this in the preview menu, which says model not trained.

and then the next problem is, when I refresh the browser on the train menu it appears "Training in progress. Checkpoint saved for epoch 27...". even though I have finished training on Google Colab.

please help, thank you
![image](https://github.com/user-attachments/assets/f5a10c1e-ef3c-4c3b-aaf7-18b5dcc45357)
![image](https://github.com/user-attachments/assets/843e513a-d7c7-46b0-89f8-2e115bc5de71)

**Comment by WidiaPrasetia** [comment]
@sergiuwaxmann Sorry if my question deviated from the issue.

I have subscribed to Ultralytics Hub Pro, I think after subscribing I can immediately train the model without paying any more. 
Can I make a refund by canceling my subscription?

thankyou

**Comment by sergiuwaxmann** [comment]
@WidiaPrasetia
I am not sure about a refund in this case as the information is clear in my opinion. For example, in our docs:
https://docs.ultralytics.com/hub/pro
https://docs.ultralytics.com/hub/cloud-training
However, I forwarded your question to our management and will get back to you as soon as possible.

**Comment by WidiaPrasetia** [comment]
okay sir @sergiuwaxmann , I will wait for further information
thank your for your response

**Comment by ruchitsolanki** [comment]
Hi @WidiaPrasetia, could you please share your HUB-associated email so we can look into the issue and take the necessary steps? We’ll review your case according to our internal policies and get back to you shortly.

**Comment by WidiaPrasetia** [comment]
@ruchitsolanki okay, this is my hub-associate email : citrabasegenep@gmail.com

**Comment by sergiuwaxmann** [comment]
@WidiaPrasetia Thank you!
